main.go

```diff
@@ -177,7 +178,7 @@ func runServer() {
 	}
 
 	// start api server
-	apiServer := api.New(cfg, mwHandler, debugUserInterface)
+	apiServer := api.New(cfg, mwHandler, debugUserInterface, *accessTokens)
```

Comment:
There's a `util.SplitTags` (that should probably be renamed) that we can use to clean-up a comma separated list and will return a `[]string` that we can then pass directly to the API. This way the API doesn't have to do any strings business and can just check if `accessTokens` is empty or not.